js/src/figure.js

```diff
@@ -634,7 +627,10 @@ var FigureView = widgets.DOMWidgetView.extend({
         this.control_orbit = new THREE.OrbitControls(this.camera, this.renderer.domElement);
         this.control_trackball.dynamicDampingFactor = 1.
         this.control_trackball.noPan = true;
-        this.control_orbit.enablePan = false;
+        // All widgets on the page would receive those keypresses all the time:
```

Comment:
I am not sure what you mean by this comment.

Reply:
I just wanted to explain why `enableKeys` is switched off:

The problem is that if there are multiple `ipyvolume` widgets on the same page and I press, e.g., the left arrow key, all of them pan to the left. Which is most likely not the expected behavior.

But I can simply remove the comment if it causes confusion.

... or replace it by whatever you prefer. Just tell me.